spyder/plugins/editor/plugin.py

```diff
@@ -274,38 +286,43 @@ def enable_tabwidth_spin(index):
         is_pep8 = codeanalysis.get_checker_executable(
                 'pycodestyle') is not None
         pyflakes_box = newcb(_("Real-time code analysis"),
-                      'code_analysis/pyflakes', default=True,
-                      tip=_("<p>If enabled, Python source code will be analyzed "
-                            "using pyflakes, lines containing errors or "
-                            "warnings will be highlighted.</p>"
-                            "<p><u>Note</u>: add <b>analysis:ignore</b> in "
-                            "a comment to ignore code analysis "
-                            "warnings.</p>"))
+                             'code_analysis/pyflakes', default=True,
+                             tip=_("<p>If enabled, Python source code will be "
+                                   "analyzed "
+                                   "using pyflakes, lines containing errors "
+                                   "or warnings will be highlighted.</p>"
+                                   "<p><u>Note</u>: "
+                                   "add <b>analysis:ignore</b> in "
+                                   "a comment to ignore code analysis "
+                                   "warnings.</p>"))
         pyflakes_box.setEnabled(is_pyflakes)
         if not is_pyflakes:
             pyflakes_box.setToolTip(_("Code analysis requires pyflakes %s+") %
                                     codeanalysis.PYFLAKES_REQVER)
         pep8_box = newcb(_("Real-time code style analysis"),
-                      'code_analysis/pep8', default=False,
-                      tip=_("<p>If enabled, Python source code will be analyzed "
-                            "using pycodestyle, lines that are not following PEP8 "
-                            "style guide will be highlighted.</p>"
-                            "<p><u>Note</u>: add <b>analysis:ignore</b> in "
-                            "a comment to ignore style analysis "
-                            "warnings.</p>"))
+                         'code_analysis/pep8', default=False,
+                         tip=_("<p>If enabled, Python source code will be "
+                               "analyzed "
+                               "using pycodestyle, lines that are not "

```

Comment:
-> "analyzed using pycodestyle, *and* lines that are not" (typo present in original)